packages/ui/src/views/labels/components/label-cell-content.tsx

```diff
@@ -1,45 +1,46 @@
 import { FC } from 'react'
 
 import { Accordion, Icon, LabelMarker } from '@/components'
-import { ILabelType, LabelValuesType } from '@/views'
+import { ILabelType, LabelValueType } from '@/views'
 import { cn } from '@utils/cn'
 
 export interface LabelCellContentProps {
   label: ILabelType
-  values: LabelValuesType
+  values?: LabelValueType[]
 }
 
 export const LabelCellContent: FC<LabelCellContentProps> = ({ label, values }) => {
-  const value = values?.[label.key]
-  const length = value ? value.length : 0
+  const isWithValues = values?.length && values.length > 0
```

Comment:
Can be simplified a bit. Not a blocker in any way
```suggestion
  const isWithValues = !!values?.length
```